index.html

```diff
@@ -115,6 +114,46 @@ <h2>Event Schedule</h2>
           <p>Don't worry if you can't make a talk, you'll have the opportunity to catch up with any talks you miss after they've been uploaded.</p>
         </div>
 
+        <div class="sub-heading">
+          <h3>
+            Workshops
+          </h3>
+        </div>
+
+        <!-- Workshops -->
+        <div class="tab-content row justify-content-center" style="padding-bottom: 30px;">
+
+
+          <div role="tabpanel" class="col-lg-9 tab-pane fade show active" id="workshops">
+
+            <div class="row schedule-item">
+              <div class="col-md-3"><time>3:00pm - 6:00pm GMT Thursday 3rd December</time></div>
+              <div class="col-md-9">
+                <h4>re:Crux workshop by JUXT</h4>
+                <p>get your tickets
+                  <a href="https://www.eventbrite.com/e/crux-workshop-reclojure-tickets-129935746315">here</a>
```

Comment:
I particularly don't like "here" s a link and prefer something more semantic like "at Eventbrite". Or just "grab your ticket" as a link.

Reply:
fixed